locationConfig.json

```diff
@@ -16,7 +16,7 @@
         "details": {
             "connector": {
                 "sproxyd": {
-                    "bootstrap": ["localhost:8182"]
+                    "bootstrap": ["localhost:8182:/proxy/arc/"]
```

Comment:
I don't see where this new config element is parsed.
Otherwise, I don't think this is the right place to put the base path: it ought to be the same path for all bootstraps (otherwise bad things will happen) so it should be in its own configuration variable IMO.